diff --git a/chrome/browser/ui/omnibox/omnibox_edit_model.cc b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
index 358f0d7520..cb300be0f6 100644
--- a/chrome/browser/ui/omnibox/omnibox_edit_model.cc
+++ b/chrome/browser/ui/omnibox/omnibox_edit_model.cc
@@ -102,6 +102,7 @@
 #include "components/strings/grit/components_strings.h"
 #include "components/vector_icons/vector_icons.h"
 #include "content/public/browser/navigation_handle.h"
+#include "device/vr/buildflags/buildflags.h"
 #include "extensions/buildflags/buildflags.h"
 #include "extensions/common/extension_features.h"
 #include "net/cookies/cookie_util.h"
@@ -538,6 +539,10 @@ bool OmniboxEditModel::ShouldShowCurrentPageIcon() const {
 
 ui::ImageModel OmniboxEditModel::GetSuperGIcon(int image_size,
                                                bool dark_mode) const {
+  if ((true)) {
+    return ui::ImageModel::FromVectorIcon(vector_icons::kGoogleColorIcon,
+                                          gfx::kPlaceholderColor, image_size);
+  }
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   if (dark_mode) {
     return ui::ImageModel::FromVectorIcon(
@@ -684,6 +689,8 @@ void OmniboxEditModel::Revert() {
 }
 
 void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
+  auto pasted = SetInputIsPastedFromClipboard(
+      controller_, paste_state_ != PasteState::kNone);
   const std::u16string input_text = MaybePrependKeyword(user_text_);
 
   // This method currently only works when there's a view, but ideally the
@@ -732,6 +739,9 @@ void OmniboxEditModel::StartAutocomplete(bool prevent_inline_autocomplete) {
 }
 
 bool OmniboxEditModel::CanPasteAndGo(const std::u16string& text) const {
+  if (IsBraveCommanderCommand(text)) {
+    return false;
+  }
   return searchbox::CanPasteAndGo(controller_->client(), text);
 }
 
@@ -1949,11 +1959,16 @@ gfx::Image OmniboxEditModel::GetMatchIcon(const AutocompleteMatch& match,
     }
   }
 
+#if (!BUILDFLAG(IS_ANDROID) || BUILDFLAG(ENABLE_VR)) && !BUILDFLAG(IS_IOS)
   bool is_starred_match = IsStarredMatch(match);
   const auto& vector_icon_type = match.GetVectorIcon(is_starred_match, turl);
 
   return controller_->client()->GetSizedIcon(vector_icon_type,
                                              vector_icon_color);
+#else
+  // No vector icons on Android without VR (AutocompleteMatch::GetVectorIcon).
+  return gfx::Image();
+#endif
 }
 
 gfx::Image OmniboxEditModel::GetMatchIconIfExtension(
